rapyuta_io_sdk_v2/config.py

```diff
@@ -69,22 +69,25 @@ def from_file(cls, file_path: str = None) -> "Configuration":
                 auth_token=data.get("auth_token"),
             )
 
-    def get_headers(self, with_project: bool = True) -> dict:
+    def get_headers(self, **kwargs) -> dict:
         """Get the headers for the configuration.
 
         Args:
-            with_project (bool): Include project guid in headers. Default is True.
+            **kwargs: Arbitrary keyword arguments.
 
         Returns:
             dict: Headers for the configuration.
         """
+        with_project = kwargs.get("with_project", True)
```

Comment:
todo: We can keep the keyword parameters explicit here. Just the above layer can pass **kwargs. So that unwanted kwargs will throw exception and we won't have to do kwargs.get.